themes/src/main/resources/theme/keycloak-preview/account/resources/app/App.tsx

```diff
@@ -63,11 +63,17 @@ export class App extends React.Component<AppProps> {
         // globally set up locale for date formatting
         moment.locale(locale);
 
+        const avatar = (
+            <div className="pf-l-toolbar__group" style={{marginLeft: '10px'}}>
+                <span id="loggedInUser">{this.kcSvc.userName}</span>
```

Comment:
I think we should display the logged in user on Welcome Screen as well. This way it's not consistent.

Reply:
It's already not consistent as it's also missing the avatar image on the welcome screen.

Reply:
Good catch with the avatar image. Strictly speaking it should be present there as well but it might be out of scope of this PR – it's up to you.
Since the Welcome Screen allows user to log in (without redirecting to a React page), there should be at least the logged in user present there, at the very least.

Reply:
I'm not sure we need the avatar image.  The image is only cosmetic.  It doesn't add any functionality and it does take up valuable space.  

@vmuzikar @edewit WDYT?

Reply:
And I could go either way.  Aesthetics are important too.

Reply:
@ssilvert I believe there are currently no plans for actually supporting avatar images, right? In that case +1 for removing the fake one as it might be confusing for the user – it implies there's some avatar but user can't change it.